xml/src/main/java/org/ehcache/xml/CacheManagerServiceConfigurationParser.java

```diff
@@ -34,4 +34,8 @@
   URI getNamespace();
 
   ServiceCreationConfiguration<T> parseServiceCreationConfiguration(Element fragment);
+
+  Class<T> getServiceCreationConfigurationType();
```

Comment:
This is type to return the service type and not the configuration type... which did you want?